tools/lybic-sandbox.py

```
@@ -27,28 +40,20 @@ def _invoke(self, tool_parameters: dict[str, Any]) -> Generator[ToolInvokeMessag
             yield self.create_json_message({
                 "status": "success",
-                "result": str(result) if result is not None else "action has no return value"
+                "result": asyncio.run(execute_action(org_id=org_id,
+                                                     api_key=api_key,
+                                                     endpoint=endpoint,
+                                                     sandbox_id=sandbox_id,
+                                                     action=action,
+                                                     provider=provider))
             })
         except Exception as e:
             raise Exception(f"Error executing action: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)